src/main/webapp/form.html

```diff
@@ -16,15 +16,14 @@ <h3>a recommendation system for students, by students!</h3>
       </div>
     </div>
     <form name="user-details" action="/user" method="POST">
-      <p>Hello! my name is</p>
-      <input type="text" name="name" class="name">
-      <br><br>
```

Comment:
Nit: Instead of padding with `br` tags, It's more appropriate to use `padding`/`margin` CSS properties :D

Reply:
right, thank you!